readthedocs/projects/utils.py

```diff
@@ -56,3 +55,13 @@ class Echo:
     def write(self, value):
         """Write the value by returning it, instead of storing in a buffer."""
         return value
+
+
+def get_projects_last_owner(user):
+    """Get projects where `user` is the last owner."""
+    from readthedocs.projects.models import Project
+    return (
+        Project.objects
```

Comment:
I did look at the SQL for this query and it looks right. However, this is probably worth having a test!